qlty-coverage/src/ci.rs

```diff
@@ -45,13 +47,29 @@ pub trait CI {
     fn build_url(&self) -> String;
 
     fn metadata(&self) -> CoverageMetadata {
+        let ci_action_version = std::env::var(QLTY_CI_ACTION_VERSION).ok();
+        let mut uploader_tool = None;
+        let mut uploader_tool_version = None;
+
+        if let Some(version) = ci_action_version {
+            let mut parts = version.split('@');
```

Comment:
Rather than doing parsing/splitting of this value in the CLI, for simplicity, could we arrange it so the CLI can just pass through the strings without needing to inspect them?

Basically similar to a "user-agent" string which is mostly opaque